agent/rpc/peering/service.go

```diff
@@ -38,6 +38,7 @@ var (
 	errPeeringTokenEmptyServerAddresses = errors.New("peering token server addresses value is empty")
 	errPeeringTokenEmptyServerName      = errors.New("peering token server name value is empty")
 	errPeeringTokenEmptyPeerID          = errors.New("peering token peer ID value is empty")
+	errPeeringBackendNil                = errors.New("peering backend was not initialized")
```

Comment:
This sounds like it should be a panic or an error defined at the validation level. I don't think it belongs with the other sentinel errors here since it's not a class of errors where a caller can fix it

Reply:
Ah RB also mentioned it below